frontend/src/Buildings/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  View, Text, ScrollView,
+  StyleSheet, ActivityIndicator,
+} from 'react-native';
+// import { CardView } from '../common/components';
```

Comment:
always delete commented out code (and useless comments too in the future)
function handlePress() {
     // TODO for one building
   }
for example this is OK (although I think unnecessary), but the commented out import or actual code is not OK.
 
```suggestion
```